src/app/src/main/java/com/example/bookmark/util/DrawerProvider.java

```diff
@@ -18,7 +18,6 @@
 import com.mikepenz.materialdrawer.model.PrimaryDrawerItem;
 
 /**
- * This class implement a navigation drawer that is passed to
  * NavigationDrawerActivity to be used in all navigation drawer activities
```

Comment:
I'm not sure you meant to delete the first line of this comment, as it doesn't really make sense anymore.